packages/utils/rollup.config.mjs

```diff
@@ -11,10 +11,10 @@ import cleanup from "rollup-plugin-cleanup";
  * @type {import('rollup').RollupOptions}
  */
 export default {
-  input: "lib/utils/index.ts",
+  input: "lib/index.ts",
   output: [
     {
-      file: "dist/utils/index.js",
+      file: "dist/index.js",
```

Comment:
I think this should be a `.ts` since it will be used internally by typescript

Reply:
Because a `.d.ts` file is supplied along with the `js` file being output, I think this means it can be supported by vanilla javascript projects and typescript. If we only supplied a `ts` output, I don't believe vanilla javascript projects would be able to use the module